docker/tools/Dockerfile

```diff
@@ -65,6 +80,7 @@ LABEL name="kanister-tools" \
     description="Tools for application-specific data protection"
 
 COPY --from=builder /restic/restic /usr/local/bin/restic
+COPY --from=builder /gosu/gosu /usr/local/bin/gosu
```

Comment:
Where will this be useful?

Reply:
There is a vulnerability in the `gosu` binary shipped with the base image and the update is not in the debian repository.